### ISSUE TITLE
PLUTO major: Data Test for Changing Row Total

### ISSUE THREAD

**Issue by sf-dcp** [NYCPlanning/data-engineering#1239]
Add data test that will check that total # of rows changed between PLUTO versions.

**Comment by fvankrieken** [comment]
What's the output of this test? Is this comparison meant to be displayed in the QA app, or is there some additional logic as well?

**Comment by sf-dcp** [comment]
@fvankrieken Nope, a super simple dbt check comparing row count of previous pluto and new pluto. There is already a similar check for pluto minor where rows shouldn't change

**Comment by fvankrieken** [comment]
And we fail if row count doesn't change? Just sort of thinking out loud here - it seems unlikely that row count wouldn't change between dtms but it doesn't seem impossible

**Comment by sf-dcp** [comment]
> And we fail if row count doesn't change? Just sort of thinking out loud here - it seems unlikely that row count wouldn't change between dtms but it doesn't seem impossible

Yes, per AD, it should fail if it doesn't change, so DE can investigate.
